pkg/routing/kvstore/kvstore.go

```diff
@@ -17,7 +18,8 @@ import (
 )
 
 const (
-	DefaultBucketName = "nodes"
+	V130BucketName    = "nodes"
+	DefaultBucketName = "node_states"
```

Comment:
> The alternative is wrapping each entry with an envelope that contains metadata about the payload

Of all options, I like this one the best, but understand it will take time to implement. 

> how about we version the buckets

Yeah, this sounds good - will address.

Reply:
done.